schemas/spdx-schema.json

```diff
@@ -380,6 +385,14 @@
             "description" : "The base name of the package file name. For example, zlib-1.2.5.tar.gz.",
             "type" : "string"
           },
+          "packagePurposes" : {
```

Comment:
I'm concerned that making PackagePurpose an array will impact fidelity when translating to other formats, plus it raises ambiguity as to the "primary purpose" of a package.

Reply:
Since this would change the main body of the spec, I opened issue #720 to track this